### PR TITLE
Added a ansible script to get cvmfs running and connected to Galaxy

### PR DESCRIPTION
**Pull request by Slugger70**
Not sure if we want to put the ansible scripts in with everything else or not...

### CODE REVIEW

File: sessions/10-uwsgi/ex1-uwsgi.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ Then, save and quit your editor.
 
 ## Section 4 - Configure reverse proxy and test
```
Comment (erasche) [inline sessions/10-uwsgi/ex1-uwsgi.md]: I think this file can be deleted, it's in the ansible for installing galaxy now. I'll make sure to pull over any missing content.

Comment (erasche) [reply sessions/10-uwsgi/ex1-uwsgi.md]: Did in my PR, nevermind.

**Comment by martenson** [comment]
not sure how my commits got here given they are in the target branch already

**Comment by martenson** [comment]
superseded by https://github.com/galaxyproject/dagobah-training/pull/71

**Pull request closed without merging**